mobile/android/android-emulator-services.ts

```diff
@@ -117,7 +117,7 @@ class AndroidEmulatorServices implements Mobile.IAndroidEmulatorServices {
 			// unlock screen
 			await this.unlockScreen(emulatorId);
 		} else {
-			this.$errors.fail("Could not find an emulator image to run your project.");
+			this.$errors.fail(`Could not find an emulator image or identifier to run your project. Please run: "tns device <platform> --available-devices"`);
```

Comment:
this code is used in different CLIs. In case you want to specify command that the user should execute, you can use something like:
```TypeScript
his.$errors.fail(`Could not find an emulator image or identifier to run your project. Please run: "${this.$staticConfig.CLIENT_NAME} device <platform> --available-devices"`);

```